tests/gocase/integration/cli_options/cli_options_test.go

```diff
@@ -0,0 +1,39 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package clioptions
+
+import (
+	"context"
+	"testing"
+
+	"github.com/apache/incubator-kvrocks/tests/gocase/util"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCLIOptions(t *testing.T) {
+	srv := util.StartServerWithCLIOptions(t, map[string]string{}, []string{"--maxclients", "23333"})
```

Comment:
I suggest that we have a prefix to overwrite config values, like `-Dmaxclients` or use another metavar to avoid conflict with common prefixes.

Reply:
Emmm...But Redis just use this syntax. Let me think of it.

Reply:
What's your opinion on this topic? cc @git-hulk @caipengbo @PragmaTwice @torwig @tanruixiang

Reply:
@tisonkun In my opinion, it's OK to use the _same_ option name without any special prefix because one value _overwrites_ another value of the _same_ option.

Reply:
I prefer no prefix. There is only 3 other non-config options (help, version, config), and will NOT increase.

Reply:
From my side, it can make Redis users happy if we obey the Redis way, so the current implementation is ok to me.

Reply:
I think if we do encounter conflicting scenarios, we can again consider making some changes.